### DISCUSSION TITLE
That's why? 求大佬，这是为什么？

### DISCUSSION THREAD

**Discussion by max-fmx** [gkd-kit/gkd#635] [Q&A] [answered]
![Screenshot_2024-06-10-11-44-32-42_befd3b55c4270226022f86e9f9cafb33](https://github.com/gkd-kit/gkd/assets/138010396/27830244-5caf-4529-a77e-db12d62e4de1)
为什么会禁用 Why is it disabled?

**Comment by AIsouler** [comment, marked as answer]
因为有的应用不需要全局规则，为了减少误触概率，所以内置禁用了
